.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+
+python:
+  - "2.7"
+
+env:
+  - VERSION="7.0" LINT_CHECK="1"
+  - VERSION="7.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - VERSION="7.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
+  - pip install geojson Shapely
+  - sudo apt-get -q -y install postgis
```

Comment:
https://github.com/OCA/geospatial/blob/8.0/__unported__/base_geoengine/README

For 7.0 I'm not sure what would be the best as modules don't support `README.rst`.
So It could wait for 8.0 version to merge `__openerp__.py` description and `README` into an `README.rst` file 

However, it is a good idea to give more visibility for those requirements in 7.0.